src/components/calcite-chip/calcite-chip.tsx

```diff
@@ -13,26 +13,26 @@ export class CalciteChip {
   //  Public Properties
   //
   //--------------------------------------------------------------------------
-  @Prop() value!: string;
-
-  /** Select theme (light or dark) */
-  @Prop({ reflect: true }) theme: "light" | "dark";
 
-  /** specify the scale of the chip, defaults to m */
-  @Prop({ reflect: true }) scale: "s" | "m" | "l" = "m";
+  /** specify the appearance style of the button, defaults to solid. */
+  @Prop({ reflect: true }) appearance: "solid" | "clear" = "solid";
```

Comment:
It would be really nice in the future if we had these types in a file somewhere. I see a lot of reuse potential for things like sizing, colors, solid/clear, etc.

Reply:
I know our app components had common types for these things.

Reply:
Yeah. Once we agree on the common types, we can extract these out.